scripts/bump_cairo_bindings.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -n "$(git status --porcelain)" ]; then
+  >&2 echo "Working tree has uncommited changes, please stash/commit them before bumping submodule version"
+  exit 1
+fi
+
+tracking_branch="$(git for-each-ref --format='%(upstream:short)' "$(git symbolic-ref -q HEAD)")"
+
+if [ $tracking_branch == "origin/master" ] ; then
+  >&2 echo "You are currently tracking master branch. Checkout to another branch before bumping submodule version"
+  exit 1
+fi
+
+
+git submodule update --recursive --remote --init cairo
+if [ -z "$(git status --porcelain)" ]; then
+  >&2 echo "Bindings are already up to date!"
+  exit 0
+fi
+
+git add ./cairo
+git commit -m "Bump cairo bindings" --no-verify
```

Comment:
This doesn't seem to be safe, we could have some staged changes already, I think we shouldn't automate this at all, I'd remove it.

Reply:
NIT

Another thing, I'd add a log like this:
```
echo "Bindings have been updated/bumped ({old_commit_hash}=>{new_commit_hash})"
```
the `*_comit_hash` thing is not necessary (command: `git rev-parse --short HEAD`)
So I think just "Bindings have been updated\bumped" is ok, the rest is just an idea.

Reply:
> This doesn't seem to be safe, we could have some staged changes already, I think we shouldn't automate this at all, I'd remove it.

i check whether working tree is clean.

Reply:
Btw not automating this will make this whole thing confusing, and we'll have problems (just like i did today).
With this script we're abstracting away how we're doing things, we just want to do a thing.

Reply:
> the *_comit_hash thing is not necessary (command: git rev-parse --short HEAD)

i'm afraid i don't understand what you mean by that

Reply:
> > This doesn't seem to be safe, we could have some staged changes already, I think we shouldn't automate this at all, I'd remove it.
> 
> i check whether working tree is clean.

Ah, right, sorry. Ok, we're good.